routes/api.js

```diff
@@ -69,6 +73,27 @@ router.get('/request_email', async (req, res) => {
     if (!body.success) {
       errors.push({ field: 'recaptcha', error: 'error_api_recaptcha_invalid' });
     }
+  } else {
+    const captcha = new Geetest({
+      geetest_id: process.env.GEETEST_ID,
+      geetest_key: process.env.GEETEST_SECRET,
+    });
+    const geetestValidate = util.promisify(captcha.validate).bind(captcha);
+    try {
+      const geetestRes = await geetestValidate(false, {
+        // eslint-disable-next-line key-spacing
```

Comment:
What's wrong here with eslint? Is it necessary to disable ESLint for these line? I dont see key-spacing error if i disable them.